$Id: fix_interwiki.patch 344 2012-06-29 15:38:56Z tg $

Fix:
* Syntax error: UPDATE IGNORE tablename is not valid SQL

--- a/dist/mediawiki-extensions-base/usr/share/mediawiki-extensions/base/Interwiki/SpecialInterwiki_body.php
+++ b/dist/mediawiki-extensions-base/usr/share/mediawiki-extensions/base/Interwiki/SpecialInterwiki_body.php
@@ -188,9 +188,9 @@ class SpecialInterwiki extends SpecialPa
 				'iw_local'  => $local, 'iw_trans'  => $trans );
 
 			if( $do == 'add' ){
-				$dbw->insert( 'interwiki', $data, __METHOD__, 'IGNORE' );
+				$dbw->insert( 'interwiki', $data, __METHOD__ );
 			} else {
-				$dbw->update( 'interwiki', $data, array( 'iw_prefix' => $prefix ), __METHOD__, 'IGNORE' );
+				$dbw->update( 'interwiki', $data, array( 'iw_prefix' => $prefix ), __METHOD__ );
 			}
 
 			if( $dbw->affectedRows() == 0 ) {
